src/Hit/Formatting.hs

```diff
@@ -0,0 +1,34 @@
+module Hit.Formatting
+  ( padLeft
+  , padRight
+  , maxLenOn
+  )
+where
+
+import qualified Data.Text                     as T
+
+-- | @padLeft n t@ pads the text 't' with spaces on the left until it reaches length 'n'.
+--
+-- @
+-- padLeft 10 "hello"' ≡ "·····hello"
+-- padLeft  3 "hello"' ≡ "hello"
+-- @
```

Comment:
We prefer multiline comments in the stye of `{- -}` not to write this redundant `--` at the beginning of each line, but that's okay to leave as it is at the moment :slightly_smiling_face:

Reply:
No problem. I'll get this fixed.